app/src/main/java/com/android/sample/ui/authentication/Login.kt

```diff
@@ -159,27 +138,14 @@ fun SignUpText(onSignUpClick: () -> Unit) {
     Text(
         modifier = Modifier.testTag("noAccountText"),
         text = "Don’t have an account yet ?",
-        style =
-            TextStyle(
-                fontSize = 14.sp,
-                fontFamily = FontFamily(Font(R.font.roboto)),
-                fontWeight = FontWeight(500),
-                color = PrimaryPurple,
-                letterSpacing = 0.14.sp))
+        style = TypographyBeatLink.bodyMedium)
 
     Spacer(modifier = Modifier.width(4.dp))
 
     // Sign up text with gradient color
     Text(
         text = "Sign up",
         modifier = Modifier.testTag("signUpText").clickable(onClick = onSignUpClick),
-        style =
-            TextStyle(
-                fontSize = 14.sp,
-                fontFamily = FontFamily(Font(R.font.roboto)),
-                fontWeight = FontWeight(500),
-                letterSpacing = 0.14.sp,
-                brush = PrimaryGradientBrush,
-                textDecoration = TextDecoration.Underline))
+        style = TypographyBeatLink.labelMedium)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
        style = MaterialTheme.typography.labelMedium)
```